src/components/Popups/PopupReply.tsx

```diff
@@ -36,34 +38,28 @@ export default function PopupReply({
     };
     const handleCancel = (event: React.MouseEvent): void => {
         handleEvent(event);
-        onCancel(text);
+        onCancel();
     };
-    const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>): void => {
-        setText(event.target.value);
-        onChange(event.target.value);
+    const handleChange = (nextEditorState: EditorState): void => {
+        onChange(nextEditorState);
     };
     const handleSubmit = (event: React.FormEvent): void => {
         event.preventDefault();
         handleEvent(event);
-        onSubmit(text);
+        onSubmit();
     };
     const handleKeyDown = (event: React.KeyboardEvent): void => {
         if (event.key !== KEYS.escape) {
             return;
         }
 
         handleEvent(event);
-        onCancel(text);
+        onCancel();
     };
     const handleFirstUpdate = (): void => {
-        const { current: textarea } = textareaRef;
-
-        if (textarea) {
-            const { value } = textarea;
-
-            textarea.focus();
-            textarea.selectionStart = value.length; // Force cursor to the end after focus
-            textarea.selectionEnd = value.length; // Force cursor to the end after focus
+        const { current: editor } = editorRef;
+        if (editor) {
+            editor.focus();
```

Comment:
Does this do the same thing as the existing code by setting the cursor to the end of the text value? Should we instead remember where the user's cursor was and set it back to the same spot?